ci-operator/step-registry/abi/conf/bm/abi-conf-bm-commands.sh

```
@@ -0,0 +1,221 @@
+        --argjson rIP "$(yq -o json '(select(
+            (.rendezvousIP | length) > 0) | .rendezvousIP
+        ) // ([
+            (.hosts[] | select(.role == "master")),
+            (.hosts[] | select(.role == "arbiter")),
+            (.hosts[] | select((.role == "") or (.role == null)))
+        ] | .[0] | [.networkConfig.interfaces[] |
+            select(.ipv4.enabled == true) |
+            .ipv4.address[0].ip
+        ] | .[0]) // error(
+            "rendezvousIP could not be determined"
+        ) ' "${OCP__ABI__CLUSTER_DIR}/agent-config.yaml")" \
```

Shellcheck (0.11.0)

[info] Make sure not to read and write the same file in the same pipeline.

(SC2094)